renderer/controller/questionset_ctrl.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+angular.module('genie-canvas')
+    .directive('templateContent', function() {
+        /**
+         * Directive to dynamically include HTML content in the existing DOM
+         * Ex:
+         *    <template-content path="/path/to/file.html"></template-content>
+         */
+        return {
+            restrict: 'E',
+            scope: false,
+            transclude: true,
+            link: function(scope, element, attrs) {
+                scope.contentUrl = attrs.path;
+                attrs.$observe('path', function(p) {
+                    scope.contentUrl = p;
+                });
+            },
+            template: '<div><button ng-click="showhint()" ng-show="currentQue.showHint" style="position:absolute; right: 2%;">hints</button></div><div ng-include="contentUrl"></div>'
+        }
+    })
+    .directive('qsGoodJob', function() {

```

Comment:
Question set already has feedback popups. Reuse/Enhance the same

**Reference:**
https://github.com/sunbird-content-plugins/org.ekstep.questionset/blob/master/renderer/utils/qs_feedback_popup.js